client/src/components/ProfilePictureGroup/ProfilePictureGroup.stories.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import ProfilePictureGroup from './ProfilePictureGroup';
+
+storiesOf('ProfilePictureGroup', module)
```

Comment:
Same here re the folder in storybook

Reply:
Added under Atoms because that's where ProfilePicture is. Since it's not strictly an Atom, maybe it should go in a Molecules folder? Didn't get a response after mentioning it was a blocker for me this morning.